.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 /node_modules
 /packages/*/node_modules
+/packages/*/LICENSE
```

Comment:
I think we should explicitly exclude `babel-parser|babel-plugin-transform-object-assign` here, e.g.
```
/packages/*/LICENSE
!/packages/babel-parser/LICENSE
!/packages/babel-plugin-transform-object-assign/LICENSE
```